actionview/lib/action_view/helpers/tag_helper.rb

```diff
@@ -26,7 +26,42 @@ module TagHelper
       PRE_CONTENT_STRINGS[:textarea]  = "\n"
       PRE_CONTENT_STRINGS["textarea"] = "\n"
 
+      class TagBuilder #:nodoc:
+        include TagHelper
```

Comment:
It is weird that a class inside this module is including itself. Can we just move what is needed in `TagHelper` inside this class and just delegate it in `TagHelper`?